### ISSUE TITLE
[ACCESSIBILITY] Event favorite saved modal not announced via screen reader 

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#290]
Describe the accessibility issue
Users of assistive technology (screen readers) are unable to know the screen is changed to a modal after selecting an event or item as a favorite. 

To Reproduce
1. Enable Talkback or voiceover on mobile device (this instance was Android Screen reader on Samsung S9 Galaxy)
2. navigate to an event detail screen on IL app
3. Select the favorites star to add even to favorites

Expected behavior
Screen reads "Add to favorites button, double tap to activate"
Once Screen reader user double taps there is a modal that appears (seemingly this only happens the very first time a user accesses the "Favorite" feature, meaning you must clear cache and data to perform this test.)
This user is not alerted to the modals presence and will be unsure of what action to take next. 
Screenshots
![Your Phone 6_14_2021 3_35_50 PM](https://user-images.githubusercontent.com/82892873/121955999-39605d00-cd26-11eb-971d-0dd9c61e72ca.png)

Smartphone (please complete the following information):

Samsung Galaxy S9 Android version 10 

Additional context
https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html

**Comment by TodorBachvarov** [comment]
This is caused by a flutter limitations. Here is an answer for autofocus issue: " TalkBack doesn't change focus unless the previous element disappears (usually during a route-like event). In the case of Modal Dialog, the previous elements are still available. For example, we don't auto focus snackbars either, even if they have an action.". So the user will have to swipe to the next element(go next) in order to focus the modal dialog.

**Comment by acovher2** [comment]
Thank you. Verified and closing issue.
